mautic/mautic
# Change report download link message

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
We can change `report_export_max_filesize_in_bytes` parameter in local.php  to 0 to force send download link to report instead of attachment.  Then message should be more universal



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Setup `report_export_max_filesize_in_bytes` in local.ph to 0
3. Send scheduled report now and then call `php  bin/console mautic:reports:scheduler`
4. Check If you receive email without phrase `File is too big to send via email, `




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11342"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
